packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,34 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import { stripUnit } from 'polished';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+export const StyledBreadcrumbItem = styled.li.attrs(() => ({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
+  'data-garden-version': PACKAGE_VERSION
 }))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  ${props => `
+    line-height: ${(props.theme.space.base * 5) / stripUnit(props.theme.fontSizes.md)}}
+    color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'}
+    & a {
+      &:any-link {
+        color: ${props.isCurrent ? 'inherit' : ''};
```

Comment:
There isn't a need to do an `else` return of an empty string. Styled-components will remove this rule if undefined/false is returned. Change to:

``
${props.isCurrent && 'inherit'}
```

Reply:
Looks like styled-components isn't removing the style.

<img width="1184" alt="Screen Shot 2019-11-28 at 10 52 33 AM" src="https://user-images.githubusercontent.com/1811365/69827708-61807d80-11cd-11ea-8faa-779eca562c86.png">